spec/requests/api/v1/api_v1_family_requests_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'rails_helper'
+
+RSpec.describe "API::V1::FamilyRequests", type: :request do
+  describe "POST /api/v1/family_requests" do
+    let!(:organization) { create(:organization) }
+    let!(:partner) { create(:partner, organization: organization) }
+    let(:request_items) do
+      Item.all.pluck(:id).sample(3).collect do |k|
+        {
+          "item_id" => k,
+          "person_count" => rand(3..10)
+        }
+      end
+    end
+
+    context "with a valid API key" do
+      subject do
+        headers = {
+          "ACCEPT" => "application/json",
+          "Content-Type" => "application/json",
+          "X-Api-Key" => ENV["PARTNER_KEY"]
```

Comment:
This potentially makes the test brittle, since it relies on external data -- it might be better to stub out the verification, like this:
```ruby
allow_any_instance_of(RequestsController).to receive(:valid_api_key?).and_return(true)
```

That can be put into the `before` block for any test block that needs to have a valid API key in the request.